recipes/flang-activation/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = "19.1.0.rc3" %}
+
+package:
+  name: flang-activation
+  version: {{ version }}
+
+build:
+  number: 0
+  # intentionally only windows (main target) & linux (debuggability)
+  skip: true  # [osx]
+
+outputs:
+  - name: flang_impl_{{ cross_target_platform }}
+    script: install_flang_impl.sh   # [unix]
+    # we don't cross-compile on windows -> no symlinks
+    requirements:
+      - flang ={{ version }}
+      - lld ={{ version }}          # [win]
```

Comment:
Do we need the exact version here?

Reply:
Should be fine with any recent lld I guess.